CryptoPrices/Sources/Data/Sources/NetworkExtension/CoinRoute.swift

```diff
@@ -24,6 +25,7 @@ extension CoinRoute: Route {
     public var path: String {
         switch self {
         case .myCoins, .trendingCoins: return "/coins/markets"
+        case .chart(let parameters): return "/coins/\(parameters.id)/market_chart"
```

Comment:
For consistency, let's use the following suggestion:
```suggestion
        case let .chart(parameters): return "/coins/\(parameters.id)/market_chart"
```

Reply:
Fixed in [4338b7e](https://github.com/nimblehq/nimble-crypto-ios/pull/75/commits/4338b7e9fe2efa557c9c644bfc0289722e2561ce) 🚀